api/src/main/scala/hmda/api/protocol/processing/EditResultsProtocol.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.api.protocol.processing
+
+import hmda.api.model._
+import hmda.validation.engine._
+import spray.json.{ DefaultJsonProtocol, DeserializationException, JsString, JsValue, RootJsonFormat }
+
+trait EditResultsProtocol extends DefaultJsonProtocol {
+
+  implicit object ValidationErrorTypeJsonFormat extends RootJsonFormat[ValidationErrorType] {
```

Comment:
This `ValidationErrorTypeJsonFormat` converter doesn't get used in the tests. Is it an artifact of a previous strategy (and should be removed)? or does it have a use case that isn't covered by tests?